flow-typed/npm/redux-loop_v4.x.x.js

```diff
@@ -0,0 +1,176 @@
+// @flow
+
+import type { ActionCreator, StoreEnhancer, Store } from "redux";
+
+declare module "redux-loop" {
+  declare export type StoreCreator = <S, A>(
+    reducer: LoopReducer<S, A>,
+    preloadedState: S,
+    enhancer: StoreEnhancer<S>
+  ) => Store<S>;
+
+  declare export type StoreCreatorExplicit<S, A> = (
```

Comment:
what is the difference between storecreator and storecreatorexplicit?

Reply:
- `StoreCreator = <S, A>` will try to guess type from the context
- `StoreCreatorExplicit<S, A>` requires types explicitly

First one doesn't always work, so the second one is "plan b" option